src/formats/apple-notes.ts

```diff
@@ -287,10 +287,12 @@ export class AppleNotesImporter extends FormatImporter {
 						AND z_pk = ${id} 
 				`;
 				
+				const filename = row.ZFALLBACKIMAGEGENERATION ? 'FallbackImage.png' : `${row.ZIDENTIFIER}.jpg`;
```

Comment:
I'm not sure when `ZFALLBACKIMAGEGENERATION` is valid, but I just wanted to double check this condition is not backwards?

Reply:
It's not.